src/cargo/ops/cargo_package.rs

```diff
@@ -40,24 +40,24 @@ pub struct PackageOpts<'cfg> {
 const ORIGINAL_MANIFEST_FILE: &str = "Cargo.toml.orig";
 const VCS_INFO_FILE: &str = ".cargo_vcs_info.json";
 
-struct ArchiveFile {
+pub struct ArchiveFile {
```

Comment:
I'd recommend adding more doc comments if we want to make items public. We can hold off this with a separate PR though.